Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/ThermoFluid/package.mo

```diff
@@ -0,0 +1,19 @@
+within Buildings.Electrical.AC.ThreePhasesBalanced.Loads.MotorDrive;
+package ThermoFluid "Package with HVAC equipment models"
+    extends Modelica.Icons.Package;
+
+
```

Comment:
empty lines.

Reply:
Thank you for reviewing these models, I have removed the unnecessary empty lines in all related files.